src/main/java/com/suse/saltstack/netapi/client/SaltStackClient.java

```diff
@@ -165,8 +180,10 @@ public SaltStackJob startCommand(String target, String function, List<String> ar
 
         // Connect to the minions endpoint and send the above lowstate data
         SaltStackResult<List<SaltStackJob>> result = connectionFactory
-                .create("/minions", SaltStackParser.JOB,  config).getResult(jsonArray.toString());
+                .create("/minions", SaltStackParser.JOB, config).getResult(
+                        jsonArray.toString());
```

Comment:
This line has been rewrapped. Please only submit changes for lines that are meant to be changed!